src/Site/Controller/Page.php

```diff
@@ -29,7 +29,7 @@ public function view(
         // special case for "brochure" HTML5 homepage
         if ($pageName == 'home') {
             if ($this->_isLoggedIn) {
-                return $app->redirect('/app/projects');
+                return $app->redirect('/redirect/project');
```

Comment:
it's a secondary point to this but I think I'd remove this altogether...if someone wants to get to the homepage, they should be able to get there...logging in isn't necessarily a big problem, most people will probably already be in the app and get redirected to the login page anyway.

Reply:
I see your point, @longrunningprocess .  I'm sure there are some websites that allow the user to access the brochure homepage while logged in, but there are a good amount of web applications that I can think of that don't either.

Some examples:

**Gmail** mail.google.com redirects to the brochure if you are logged out, and redirects to the app if you are logged in.
![image](https://user-images.githubusercontent.com/3444521/134147340-518f51ff-d379-45ce-a77b-9a53fa66ccda.png)

**PayPal** paypal.com redirects to a homepage if you are logged out, and takes you to the app if you are logged in.
![image](https://user-images.githubusercontent.com/3444521/134147922-a2185b8a-da12-47ff-828a-b9b9fc8a9cf5.png)

In both of these cases, the user can't click to get to these pages directly once they are logged in, however they might be able to access them directly from the URL.

A compromise might be to make a valid route called /home which shows the brochure homepage no matter what.  Then have logic something like the above that if you are logged in, you go to the app, and if not you go to the `/home` page.

Reply:
I've left this as-is; we do want someone to be able to go to languageforge.org and be taken directly to their project if their login cookie is still valid.